### ISSUE TITLE
Version 5.0.x does not see the .vue files

### ISSUE THREAD

**Issue by TokugawaTakeshi** [TypeStrong/fork-ts-checker-webpack-plugin#470]
Thank you for the plugin.

Not want to say what I has setup `fork-ts-checker-webpack-plugin` for vue application exactly as in [documentation](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin#vuejs) because this plugin must not force the settings like `target` or `sourceMap`, but I fulfilled to two necessary conditions (`TypeScript >= 3.8.0` and `"importsNotUsedAsValues": "preserve"`) .

## Current behavior

1. For ".ts" files, all ESLint warnings and errors are being displayed.
2. Manual execution of ESLint for ".vue" files works, therefore requirements for TypeScript-ESLint has been been provided properly.
3. **This issue's subject**: ESLint violations in Vue files is not being output to console.

## Expected behavior

In **3**, ESLint violations in Vue files are being output to console.

## Issue reproduction repository

1. Clone this repository (will be deleted after issue closed): https://github.com/TokugawaTakesi/fork-ts-checker-webpack-plugin-issue-470
2. Install all dependencies and run building with ESLint output: `npm i & npm run build`

You will be ESLint output only for `EntryPoint.ts`, but not for `Application.vue`.
You can run manually run ESLint for .vue file and confirm that ESLint see this errors:

```
eslint SPA_Stage-SourceFiles/Application.vue
```

## Environment
 - **fork-ts-checker-webpack-plugin**: 5.0.5
 - **typescript**: 3.9.5
 - **eslint**: 6.8.0
 - **webpack**: 4.43.0
 - **os**: Windows 10

### tsconfig.json

```json
{
  "compilerOptions": {

    "target": "ES2017",

    "module": "CommonJS",
    "moduleResolution": "node",
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,

    "sourceMap": true,

    "experimentalDecorators": true,
    "skipLibCheck": true,

    "strict": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,

    "importsNotUsedAsValues": "preserve",

    "baseUrl": "./",
    "paths": {
      "@ProjectRoot/*" : [ "./*" ]
    }
  }
}
````

### Webpack config (could be found in gulpfile.js):

```javascript
{
  // ...
  
  module: {
    rules: [
      {
        test: /\.ts$/u,
        loader: "ts-loader",
        options: {
          appendTsSuffixTo: [ /\.vue$/u ],
          transpileOnly: true
        }
      },
      {
        test: /\.vue$/u,
        loader: "vue-loader"
      },
      // ...
    ]
  },
  resolve: {
    extensions: [ ".mjs", ".ts", ".js" ],
    alias: {
      vue: "vue/dist/vue.common.js",
      "@ProjectRoot": Path.resolve("")
    }
  },

  devtool: isDevelopmentBuildingMode ? "eval-cheap-source-map" : false,
  plugins: [
    new ForkTsCheckerWebpackPlugin({
      typescript: {
        configFile: Path.resolve(process.cwd(), "tsconfig.json"),
        extensions: {
          vue: true
        }
      },
      eslint: {
        enabled: true,
        files: "./**/*.ts"
      }
    }),
    new VueLoaderPlugin()
  ]
};
```

### .eslintrc.yaml

```yaml
parser: vue-eslint-parser
parserOptions:
  parser: "@typescript-eslint/parser"
  sourceType: module
  project: tsconfig.json
  tsconfigRootDir: ./
  extraFileExtensions: [ ".vue" ]

env:
  es6: true
  browser: true
  node: true

plugins:
  - "@typescript-eslint"
  - vue


rules: # ....
```

**Comment by piotr-oles** [comment]
Hi!
I think the issue is that you include only `*.ts` files in the configuration:
```
      eslint: {
        enabled: true,
        files: "./**/*.ts"
      }
```

Please try to change it to:
```
      eslint: {
        enabled: true,
        files: ["./**/*.ts", "./**/*.vue"]
      }
```

**Comment by kevinmarrec** [comment]
Sounds to be duplicated of #452 , fixed in `5.0.6` released 1 hour ago :)

/cc @piotr-oles 

EDIT : More likely I'm wrong as `eslint` issue may be totally out of context with #452

**Comment by TokugawaTakeshi** [comment]
It works! Thank you for quick response!
